src/index.jsx

```diff
@@ -3,9 +3,13 @@ import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./App";
 
+import { ChakraProvider } from "@chakra-ui/react";
```

Comment:
noticed you like to leave a space in-between 3rd party packages and imported components? can we switch these around to follow what you've done on the other files

```suggestion
import { ChakraProvider } from "@chakra-ui/react";

import App from "./App";
```

Reply:
Tidied all the other files too! 👍  [20f1fbd](https://github.com/8rianHo/vietnamese-directory/pull/2/commits/20f1fbdf56d23a28c314773f7a0974040eadce8c)

Reply:
nerd